### ISSUE TITLE
Support for workspace/didChangeWorkspaceFolders?

### ISSUE THREAD

**Issue by sajinieKavindya** [angelozerr/lsp4xml#360]
I am currently building an VSCode extension to support Synapse language by extending lsp4xml and I am new to this subject. For some of the functionalities in my extension, I need to monitor the workspace folder change event and maintain the workspace folder's names from the language server. However, I noticed that the lsp4xml has no support for workspace/didChangeWorkspaceFolders. 

Any plans to add support for $subject?

Thanks.

**Comment by angelozerr** [comment]
> However, I noticed that the lsp4xml has no support for workspace/didChangeWorkspaceFolders.

You mean that there is not extension (with SPI) like completion, hperlink, etc provided by lsp4xml? Could you please explain more your need please.

**Comment by sajinieKavindya** [comment]
Not exactly... I am building an extension related to XML which uses **lsp4xml** as the language server. And I came up with this requirement where I want to monitor workspace folder change event from the language server side because I need to maintain a list of workspaces opened inside a VS Code instance (since VS Code has the ability to maintain multi-root workspaces). What I am expecting is the `didChangeWorkspaceFolders` method implementation (as shown below) in [here](https://github.com/angelozerr/lsp4xml/blob/b8e5a7a51bf6147e02da91eb489faa51ce49c14d/org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/XMLWorkspaceService.java#L49).

```@Override
	public void didChangeWorkspaceFolders(DidChangeWorkspaceFoldersParams params) {
		//method implementation
	}
``` 
Will you please let me know whether you are going to add this support or if there is any other way to do the task which I am expecting?

**Comment by angelozerr** [comment]
You wish to track call of didChangeWorkspaceFolders, is that? If it that lsp4xml must provide a new extension like completion, hyperlink, etc for didChangeWorkspaceFolders
